docs/platform/developing/building-all.md

```diff
@@ -57,203 +57,58 @@ To clone the latest sources of all projects over SSH:
    Initialize the repository by executing the following command:
 
    ```
-   $ repo init -u ssh://<Username>@review.tizen.org:29418/scm/manifest -b <tizen branch> -m ${profile}_${repository}.xml
+   $ repo init -u ssh://<Username>@review.tizen.org:29418/scm/manifest -b <tizen branch> -m <profile>_<repository>.xml
    ```
+   > **Note**
+   >
+   > If you encounter some problems using repo tools, please check [Setting Up the Repo Tool](./setting-up.md#setting-up-the-repo-tool).
```

Comment:
@sagarkedari, since line 194 change is suggested, could you please provide the comment here as well.

Reply:
For consistency in the format between lines 64 and 194, below are my suggestions. Kindly use the same format at both the places.

```suggestion
   > If you encounter some problems using repo tools, see [Setting Up the Repo Tool](./setting-up.md#setting-up-the-repo-tool).
```

OR

```suggestion
   > For information on installing and setting up the repo tool, see [Setting Up the Repo Tool](./setting-up.md#setting-up-the-repo-tool).
```